system-monitor/system-monitor.py

```diff
@@ -0,0 +1,101 @@
+import subprocess
+import sys 
+import time
+import threading
+from flask import Flask, request
+from MonitorThread import MonitorThread
+
+app = Flask(__name__)
+
+monitor_thread = None
+
+# {
+#     "monitor": false
+# }
+
+@app.route('/monitor', methods=['delete'])
+def stop_monitor():
+    try:
+        try:
+            data = request.get_json()
+            if not data['monitor']:
+                global monitor_thread
+                monitor_thread.stop()
+                
+        except:
+            raise ValueError
+
+    except ValueError:
+        return 'No request body / No active monitoring', 400
+    
+    return 'Monitoring stopped', 200
+        
+# {
+#     "container_name": 'name',
+#     "monitor": false
```

Comment:
do we really need this param? Isn't it obvious that we wan to monitor if we use post method? I also don't see usage of this param at all